### PR TITLE
enchancement: updated naming convention from clientSet to dynamicClient (#203)

### LINKED ISSUE #203: Can we use this variable `dynamicClient` instead to `clientSet`
Can we use this variable `dynamicClient` instead to `clientSet`

_Originally posted by @rahulchheda in https://github.com/litmuschaos/chaos-operator/pull/199_

### PR DESCRIPTION
**Pull request by nikzayn**
**What this PR does / why we need it**:
This PR updates the naming convention from **clientSet** to **dynamicClient**

**Which issue this PR fixes**
- Fixes #203 

**Special notes for your reviewer**:
- To verify the changes and check if everything is working fine. Just a sanity check.

**Checklist:**
-   [x] Fixes #203 
-   [x] PR messages has document related information

### CODE REVIEW

File: pkg/client/kubernetes/kubernetes.go [hunk 1]
```diff
@@ -27,9 +27,9 @@ func CreateClientSet() (*kubernetes.Clientset, error) {
 	if err != nil {
 		return &kubernetes.Clientset{}, err
 	}
-	clientSet, err := kubernetes.NewForConfig(restConfig)
+	dynamicClient, err := kubernetes.NewForConfig(restConfig)
```
Comment (ispeakc0de) [inline pkg/client/kubernetes/kubernetes.go:30]: Can we revert this change as this is not a dynamic client?

File: tests/bdd/bdd_test.go [hunk 2]
```diff
@@ -48,10 +48,10 @@ import (
...
 
 var (
-	kubeconfig string
-	config     *restclient.Config
-	client     *kubernetes.Clientset
-	clientSet  *chaosClient.LitmuschaosV1alpha1Client
+	kubeconfig    string
+	config        *restclient.Config
+	client        *kubernetes.Clientset
+	dynamicClient *chaosClient.LitmuschaosV1alpha1Client
```
Comment (ispeakc0de) [inline tests/bdd/bdd_test.go:54]: Can we revert this change as this is not a dynamic client? and change the rest of the places as well

**Comment by neelanjan00** [comment]
Closing this PR as it is not needed. We have closed the issue as well. Still, thanks for your contribution!
